Kinvey/Kinvey/DataStore.swift

```diff
@@ -409,8 +409,7 @@ open class DataStore<T: Persistable> where T: NSObject {
 
     /// Clear all data for the collection attached to the DataStore.
     open func clearCache() {
```

Comment:
@heyzooi  Not directly related to this change, but shouldn't `clearCache()` optionally take a query argument?